### PR TITLE
[zh] Sync #13719 into Chinese with some fixes and improvements

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync #13719 into Chinese:
```
content/zh/boilerplates/ambient-alpha-warning.md
content/zh/docs/ops/ambient/getting-started/index.md
```

Some fixes and improvements:
```
content/zh/docs/ops/ambient/architecture/index.md
content/zh/docs/ops/best-practices/image-signing-validation/index.md
content/zh/docs/ops/best-practices/observability/index.md
content/zh/docs/setup/additional-setup/config-profiles/index.md
content/zh/docs/setup/additional-setup/customize-installation/index.md
content/zh/docs/setup/additional-setup/pod-security-admission/index.md
content/zh/docs/setup/additional-setup/sidecar-injection/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/boilerplates/ambient-alpha-warning.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+---
+{{< warning >}}
+Ambient 目前处于 [Alpha 状态](/zh/docs/releases/feature-stages/#feature-phase-definitions)。
+
+请**不要在生产环境中运行 Ambient 模式**，
+并确保在使用前详细查阅[功能阶段定义](/zh/docs/releases/feature-stages/#feature-phase-definitions)。
```
Comment (windsonsea) [inline content/zh/boilerplates/ambient-alpha-warning.md:7]:
```suggestion
务必先行斟酌[特性阶段定义](/zh/docs/releases/feature-stages/#feature-phase-definitions)再行使用 Ambient。
```

File: content/zh/boilerplates/ambient-alpha-warning.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+特别是，`alpha` 版本中包含已知的性能、稳定性和安全问题。
```
Comment (windsonsea) [inline content/zh/boilerplates/ambient-alpha-warning.md:8]:
```suggestion
具体而言，`alpha` 版本意味着存在已知的性能、稳定性和安全性问题，
```

File: content/zh/boilerplates/ambient-alpha-warning.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+并且存在计划中的重大变更，包括一些会阻碍升级的更改。所有这些限制都会在升级到 `beta` 版之前被解决。
```
Comment (windsonsea) [inline content/zh/boilerplates/ambient-alpha-warning.md:9]:
```suggestion
还存在一些计划中的破坏性变更，其中某些变更可能会令升级失败。
这些是进阶至 `beta` 之前需要解决的问题。
```

File: content/zh/boilerplates/ambient-alpha-warning.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+---
+{{< warning >}}
+Ambient 目前处于 [Alpha 状态](/zh/docs/releases/feature-stages/#feature-phase-definitions)。
+
+请**不要在生产环境中运行 Ambient 模式**，
```
Comment (windsonsea) [inline content/zh/boilerplates/ambient-alpha-warning.md:6]:
```suggestion
**请勿在生产环境中使用 Ambient 模式**，
```

**Review by windsonsea — COMMENTED** [review]
原来有类似的译文时，不用重翻，又累又不一致呐 :)

File: content/zh/docs/setup/additional-setup/pod-security-admission/index.md [hunk 5]
```diff
@@ -56,9 +56,9 @@ Istio 默认会将 Init 容器 `istio-init` 注入到网格中部署的 Pod 内
     ✔ Installation complete
     {{< /text >}}
 
-## 部署样例应用{#deploy-sample-app}
+## 部署示例应用 {#deploy-sample-app}
```
Comment (my-git9) [inline content/zh/docs/setup/additional-setup/pod-security-admission/index.md:59]: 下面用的都是 `样例`，建议同一篇文章保持一致

**Comment by wilsonwu** [comment]
> 原来有类似的译文时，不用重翻，又累又不一致呐 :)

本来准备照抄，发现原文有些不同就有话了一下。

File: content/zh/docs/setup/additional-setup/pod-security-admission/index.md (hunk 5 above)
Comment (wilsonwu) [reply content/zh/docs/setup/additional-setup/pod-security-admission/index.md:59]: Got it, will fix ASAP.

Comment (wilsonwu) [reply content/zh/docs/setup/additional-setup/pod-security-admission/index.md:59]: Fixed all.

**Review by windsonsea — APPROVED** [review]
/lgtm

**Pull request merged**